src/panels/RemoveSecretPanel/Fields/Fields.tsx

```diff
@@ -53,8 +60,8 @@ const Fields = ({
         </>
       ) : (
         <p>
-          This secret has one revision ({secret?.["latest-revision"]}) and will
-          be completely removed.
+          This secret has one revision ({latestRevision}) and will be completely
```

Comment:
`latestRevision` can be `null`, so while this is very unlikely, the logic in this component could change so it might be safer to do:

```suggestion
          This secret has one revision{latestRevision ? ` (${latestRevision})` : ""} and will be completely
```

Reply:
Done. Thanks!